xml/mysql_banners.xml

```diff
@@ -998,6 +998,15 @@
     <param pos="0" name="service.product" value="MariaDB"/>
     <param pos="0" name="service.edition" value="Enterprise Edition"/>
   </fingerprint>
+  <fingerprint pattern="(\d{1,2}\.\d{1,2}\.\d{1,2})-maxscale">
+    <description>MariaDB MaxScale</description>
+    <example service.version="2.0.3">5.5.5-10.0.0 2.0.3-maxscale</example>
+    <example service.version="2.1.14">5.5.5-10.0.0 2.1.14-maxscale</example> 
```

Comment:
Are two examples really necessary? They exercise the same regex so not sure it's really necessary.